lib/signaling/v2/remoteparticipant.js

```diff
@@ -52,14 +53,18 @@ class RemoteParticipantV2 extends RemoteParticipantSignaling {
       }
     });
 
+    Object.entries(options.otherProperties).forEach(([key, value]) => {
+      Object.defineProperty(this, key, { value });
```

Comment:
I am not really understanding why we want some runtime properties on this type. If they are really known at runtime only, Could we keep them in a single Map property instead.
something like 
```
Object.defineProperties(this, {
  otherProperties:  options.otherProperties
});
```

I am afraid generating them as object properties would make it hard to convert to typescript.